README.md

```diff
@@ -44,15 +44,41 @@ To run checks against an assignment, use the `gator grade` command.
 
 #### Interpreting Output
 
-Output team
+A dictionary will come from the input team that mostly will contain list,
+unpack the dictionary, run through gatorgrade, and return as a list of tuples.
+Each tuple will contain a descriptive boolean that will either Pass or Fail.
+If Passed, it will come through as an empty string. If Failed it will come
+through as what exactly failed within the assignment.
 
 ### Generating a gatorgrade.yml file
 
-Generate team
+In order to generate a ‘gatorgrade.yml’ file, you should run the following
+command in your terminal:
+
+‘’’
+Gator grade generate
```

Comment:
Does the generate team have a definitive preference for a capitalized `Gator` or could it possibly be all lowercase similar to the `gator grade` command mentioned earlier in the readme or the `gator gradle` command that was used by `GatorGradle`?